README.md

```diff
@@ -12,7 +12,7 @@ Install
 -------
 
 - Install Go and MongoDB.
-- `go get gopkg.in/mgo.v2 && go get github.com/gorilla/mux`
+- Dependencies are managed using Go modules in `go.mod`. Go modules are available in all supported Go versions.
```

Comment:
How do I install the modules that are in `go.mod`? What is the command?

Reply:
@mlr0p add a line between 14 and 15 with `- go get github.com/zxsecurity/steamer` to resolve this :)

Reply:
@SoftPoison I don't think that downloads the modules required by the importers in the sub directories. And if I run `go get github.com/zxsecurity/steamer/importers` it will throw an error saying something like `main redeclared in this block`.
So now the best way might be to just run `go run steamer.go` or `go run importers/kickstarter.go`  because go will automatically downloads the required modules if it can't find one.